code/espurna/homeassistant.ino

```diff
@@ -123,6 +124,7 @@ void _haSendSwitches() {
             JsonObject& config = jsonBuffer.createObject();
             _haSendSwitch(i, config);
             config.printTo(output);
+            jsonBuffer.clear();
```

Comment:
This is not technically required bc buffer will be deconstructed each iteration. Just a reminder that it can be removed after serialization in done.